pkg/apis/workspaces/v1alpha1/projects.go

```diff
@@ -13,6 +13,16 @@ type Project struct {
 	ProjectSource `json:",inline"`
 }
 
+type StarterProject struct {
+	Name string `json:"name"`
+
```

Comment:
@kadel Is there a reason when you omit the `clonePath` which is a field common to all types of project sources, and instructs about where to clone the project ?

Why not simply have `starterProjects` be a list of `Project` structures ? This would avoid duplicating code, no ?

Reply:
forgot about `clonePath` :-( will fix that

> Why not simply have starterProjects be a list of Project structures ? This would avoid duplicating code, no ?

I can't do that as there is no Description field in `Project` but I can embed whole `Project` in `StarterProject`

Reply:
@davidfestal I've updated this to reuse existing `Project` struct

Reply:
Thanks !